exporter/datadogexporter/examples/host-metrics.yaml

```diff
@@ -0,0 +1,40 @@
+receivers:
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
```

Comment:
Can we also enable these three https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2367c0e95d8a36b9c21b2fba3726c6b50ba23730/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/metadata.yaml#L36C14-L60 ?